src/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj

```diff
@@ -58,7 +58,16 @@
     </EmbeddedResource>
   </ItemGroup>
   <ItemGroup>
+    <None Include="Data\CodePageNameMappings.csv" />
```

Comment:
I am fine to keep it. just confirm the files will not get copied to the output folder.